lib/jsonapi-response.js

```diff
@@ -45,46 +45,68 @@ function getRelationships (data) {
           id: ''
         };
 
-        rel.id = TypeMapping.toExternal(el.admin.uid);
-        rel.type = TypeMapping.toExternal(el.admin.id.split('-')[0]);
-
-        relationships[key].data.push(rel);
+        if (el.admin) {
+          rel.id = TypeMapping.toExternal(el.admin.uid);
+          rel.type = TypeMapping.toExternal(el.admin.id.split('-')[0]);
+          relationships[key].data.push(rel);
+        } else {
+          addAsAttribute(el, key, response);
+        }
```

Comment:
I see what you mean here, but I'd be inclined to ignore this relationship entirely if it has no ID to keep to the spec.